portfolio_website.html

```diff
@@ -17,44 +17,191 @@
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+             </li>
+             <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+            </li>
+            <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="proj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+           </li>
+            </ul>
+          </section>
+          <section class="about-section">
+            <h2 class="greetings about">About me</h2>
+            <div class="about-container img2">
+              <p class=" intro">
+                Hello I'm a software developer! I can help you build a product,
+                feature or website. Look through some of my work and experience! If
+                you like what you see and have a project you need coded, dont hestiate
+                to contact me.
+              </p>
+              <a class="btn" href="">Get My Resume</a>
+              <p class="underline"></p>
+            </div>
+            <ul>
+              <li>
+                <ul class="list-container">
+                   <li class="list first"><i class="fa fa-square icons"></i>Language</li>
```

Comment:
I think the right font family used for this text is (Font family: InaiMathi) as the project requires you stick to the design as much as possible. check [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)